documentation/examples/remote_storage/remote_storage_adapter/main.go

```diff
@@ -25,6 +25,7 @@ import (
 	"sync"
 	"time"
 
+	"github.com/blang/semver"
```

Comment:
We vendor all the other deps necessary for building the examples, so let's vendor this one too (if we really need it at all - I don't think we should be getting into the business of supporting too many old versions anyways, especially since it's all still marked experimental).